guides/source/association_basics.md

```diff
@@ -2178,8 +2179,8 @@ The `collection.where` method finds objects within the collection based on the c
 
 ##### `collection.exists?(...)`
 
-The `collection.exists?` method checks whether an object meeting the supplied
-conditions exists in the collection. It uses the same syntax and options as
+The `collection.exists?` method checks whether an object within the collection by querying the table. It will not find
```

Comment:
> It will not find objects that are not in the table.

Let's remove these, its redundant information.

Reply:
Done

Reply:
>It will not find objects that are not in the table.

This sentence is still there.